controller/tests/array_action/xiv/array_mediator_xiv_tests.py

```diff
@@ -510,3 +517,43 @@ def test_get_iscsi_targets_by_iqn_success(self):
         targets_by_iqn = self.mediator.get_iscsi_targets_by_iqn()
 
         self.assertEqual(targets_by_iqn, {"iqn1": ["1.2.3.4", "[::1]"]})
+
+    def _convert_size_bytes_to_blocks(self, size_in_bytes):
+        return self.mediator._convert_size_bytes_to_blocks(size_in_bytes)
```

Comment:
I know you just moved this existing issue, but tests should not rely on tested logic.
could you please avoid it and use hardcoded values?

Reply:
done

Reply:
wouldn't the logic you just "done" round it down?
could we use hardcoded values instead of more error-prone logic?

Reply:
ok, done